Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py

```diff
@@ -1,6 +1,1031 @@
+    if dkim_result in res_dkim:
+        res_dkim[dkim_result] += 1
+    else:
+        res_dkim['other'] += 1
+    res_dkim['non-positive'] = spf_result in set(['none', 'fail', 'softfail'])
+    list_unsubscribe = get_header_value(email_headers, header_name='List-Unsubscribe')
+    unsubscribe_post = get_header_value(email_headers, header_name='List-Unsubscribe-Post-SPF')
+    res_unsubscribe = 1 if list_unsubscribe is not None or unsubscribe_post is not None else 0
+    from_value = get_header_value(email_headers, header_name='From')
+    return_path_value = get_header_value(email_headers, header_name='Return-Path')
+    last_received_value = get_header_value(email_headers, header_name='Received', index=-1)
+    if from_value is not None:
+        from_address = re.findall(r'<\S+>', from_value)[0][1:-1]
+    else:
+        from_address = ''
+    if return_path_value is not None:
+        return_path_adress = return_path_value.strip().lstrip('<').rstrip('>').strip()
+    else:
+        return_path_adress = ''
+    if last_received_value is not None:
+        received_address_list = re.findall(r'(?<=from )\S+', last_received_value)
+        if len(received_address_list) > 0:
+            received_address = received_address_list[0]
+        else:
+            received_address = ''
+    else:
+        received_address = ''
+    try:
+        from_domain = from_address.split('@')[-1].split('.')[-2] if from_address != '' else ''
+    except Exception:
+        print('error on from_address: {}'.format(from_address))
+        from_domain = ''
+
+    if from_address == '':
+        return_path_same_as_from = float('nan')
+    else:
+        return_path_same_as_from = from_address == return_path_adress
+
+    if from_domain == '' or received_address == '':
+        from_domain_with_received = float('nan')
+    else:
+        from_domain_with_received = from_domain in received_address
+    received_res = {}
+    recevied_headers = [header_dict for header_dict in email_headers if header_dict['name'] == 'Received']
+    count_received = len(recevied_headers)
+    received_res['count_received'] = count_received
+    content_type_value = get_header_value(email_headers, header_name='Content-Type', index=0, ignore_case=True)
+    if content_type_value is not None and ';' in content_type_value:
+        content_type_value = content_type_value.split(';')[0]
+    adress_res = {
+        'return_path_same_as_from': return_path_same_as_from,
+        'from_domain_with_received': from_domain_with_received
+    }
+    res = {}
+    for k, v in res_spf.items():
+        res['spf::{}'.format(k)] = v
+    for k, v in res_dkim.items():
+        res['dkim::{}'.format(k)] = v
+    res['unsubscribe_headers'] = res_unsubscribe
+    for k, v in adress_res.items():
+        res[k] = v  # type: ignore
+    for k, v in received_res.items():
+        res[k] = v
+    res['content-type::{}'.format(content_type_value)] = 1
```

Comment:
more possible features:
- from domain alexa rank
- return path domain alexa rank
- reply to alexa rank